syft/frameworks/torch/pointers/pointer_tensor.py

```diff
@@ -219,7 +219,7 @@ def create_pointer(
                 id_at_location=id_at_location,
                 owner=owner,
                 id=ptr_id,
-                garbage_collect_data=garbage_collect_data,
+                garbage_collect_data=True if garbage_collect_data is None else garbage_collect_data,
```

Comment:
Not sure why this is different from the previous implem where the default is also False but stated in the signature